biothings/web/handlers/base.py

```diff
@@ -114,6 +119,29 @@ def _parse_args(self, reqargs):
         if not self.name:  # feature disabled
             return {}  # default value
 
+        # Check if handler defines kwargs but not a unique name - this is an error
+        if self.name == "__base__" and self.__class__ != BaseAPIHandler:
+            # Check if this handler defines its own kwargs (not inherited from BaseAPIHandler)
+            handler_has_kwargs = (
+                hasattr(self.__class__, 'kwargs') and
+                self.__class__.kwargs is not BaseAPIHandler.kwargs
+            )
+
+            if handler_has_kwargs:
+                # Handler defines kwargs but uses default name - this will cause conflicts
+                raise HTTPError(
```

Comment:
very minor: I think this is a developer-side of error, not a user-facing error, so seems a normal `ValueError` or similar would be more proper. Developer should fix it, otherwise the app won't start.